app/src/test/java/com/aws/amazonlocation/mock/NavigationDataTest.kt

```diff
@@ -79,57 +80,61 @@ class NavigationDataTest {
         navigationData.endLat = 21.5511451
         navigationData.endLng = 21.5511451
         navigationData.isDataSuccess = true
-        val result = navigationData.getRegions()
+        val result = navigationData.getAddress()
 
         assertEquals(CITY_NAME, result)
     }
 
     @Test
-    fun testGetRegionsWithDataSuccessSubRegion() {
+    fun testGetAddressWithDataSuccessSubRegion() {
+        // `getAddress()` should return `country` if both are available.
         val navigationData = NavigationData(
             subRegion = SUB_REGION_NAME,
             country = COUNTRY_NAME,
             isDataSuccess = true
         )
 
-        val result = navigationData.getRegions()
+        val result = navigationData.getAddress()
 
         assertEquals(COUNTRY_NAME, result)
     }
 
     @Test
-    fun testGetRegionsWithDataSuccessRegion() {
+    fun testGetAddressWithDataSuccessRegion() {
+        // `getAddress()` should concatenate `region, country` as the result.
         val navigationData = NavigationData(
             region = REGION_NAME,
             country = COUNTRY_NAME,
             isDataSuccess = true
         )
 
-        val result = navigationData.getRegions()
+        val result = navigationData.getAddress()
 
         assertEquals("$REGION_NAME, $COUNTRY_NAME", result)
     }
 
     @Test
-    fun testGetRegionsWithDataSuccessCountry() {
+    fun testGetAddressWithDataSuccessCountry() {
+        // `getAddress()` should return the `country` value as the address.
         val navigationData = NavigationData(
             country = COUNTRY_NAME,
             isDataSuccess = true
         )
 
-        val result = navigationData.getRegions()
+        val result = navigationData.getAddress()
 
         assertEquals(COUNTRY_NAME, result)
     }
 
     @Test
-    fun testGetRegionsWithDataFailure() {
+    fun testGetAddressWithDataFailure() {
+        // of `destinationAddress` directly, irrespective of other fields.
```

Comment:
This comment is missing a few words I think

Reply:
Removed as the same test case exists